### PR TITLE
Working on Store Info section of page

### PR DESCRIPTION
**Pull request by andrayantelo**

### CODE REVIEW

File: docs/storeinfo.html [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+        
+        <div class="page-header text-center">
+            <h1>About Us</h1>
+        </div>
+        
+    <div class="row">
+      <div class="col-xs-12 col-sm-12 col-md-offset-1 col-md-10 col-lg-offset-1 col-lg-10">
+    
+        <p class="page-text">
+            IVY'S ATTIC is the premier Furniture Store in the Englewood, Florida area.
```
Comment (johntyree) [inline docs/storeinfo.html:114]: Why is this CAPITALIZED?

File: docs/index.html [hunk 2]
```diff
@@ -341,7 +341,7 @@ <h1 class=".page-header">Available Items</h1>
                 </h4>
                 <ul class="menu-footer collapse store-info">
                     <li>
-                        <a href="#">About Us</a>
+                        <a href="/docs/aboutus.html">About Us</a>
```
Comment (johntyree) [inline docs/index.html:344]: This PR says it's merged but I don't see it when i look at the page source.

File: docs/storeinfo.html (hunk 1 above)
Comment (andrayantelo) [reply docs/storeinfo.html:114]: I copy pasted from current ivy's attic page.

File: docs/index.html (hunk 2 above)
Comment (andrayantelo) [reply docs/index.html:344]: Not using this anymore.

**Pull request merged**